app/assets/javascripts/components/Register.jsx

```diff
@@ -0,0 +1,107 @@
+class Register extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {"success": null};
+  }
+
+  FieldGroup({ id, label, help, ...props }) {
+    const { Checkbox, Radio, FormGroup, ControlLabel, FormControl, Button, HelpBlock } = ReactBootstrap;
+    return (
+      <FormGroup controlId={id}>
+        <ControlLabel>{label}</ControlLabel>
+        <FormControl {...props} />
+        {help && <HelpBlock>{help}</HelpBlock>}
+      </FormGroup>
+    );
+  }
+
+  createUser = (evt) => {
+    evt.preventDefault()
+    const inputs = evt.target.getElementsByTagName("input");
+    let upload = "";
+    if (inputs[4].files.length > 0) {
+      upload = inputs[4].files[0];
+    }
+    const payload = {
+      "email": inputs[0].value,
+      "password": inputs[1].value,
+      "first_name": inputs[2].value,
+      "last_name": inputs[3].value,
+      "avatar": upload
+    };
+    Requester.post('/api/users', payload).then((data) => {
+      // Handle success/failure.
+      if (data["message"] == "Attorney successfully created!") {
+        this.setState({"success": 1});
```

Comment:
Also i think Amy may have used toastr notifications for something. We may want to be using that instead to keep our notification system consistent

Reply:
we moved off of toastr cuz it requires jquery

Reply:
ah yes i rmemeber. ty